src/development/tools/devtools/performance.md

```diff
@@ -108,6 +118,43 @@ with the mouse wheel / trackpad
 You can click an event to view CPU profiling information in the CPU profiler
 below, described in the next section.
 
+## Enhance tracing 
+
+To view more detailed tracing in the timeline events chart, use the
+options in the enhance tracing dropdown:
+
+{{site.alert.note}}
+Frame times may be negatively affected when these options are enabled.
+{{site.alert.end}}
+
+![Screenshot of enhance tracing dropdown]({{site.url}}/assets/images/docs/tools/devtools/enhance-tracing.png)
+
+To see the new timeline events, trigger a new frame by interacting with
+your app.
+
+### Track widget builds
+
+To see the build() methods in the timeline, enable the Track Widget
+Builds option. The name of the widget is shown in the timeline event.
+
+![Screenshot of track widget builds]({{site.url}}/assets/images/docs/tools/devtools/track-widget-builds.png)
+
+### Track layouts
+
+To see layout information, enable the Track Layouts option:
+
+![Screenshot of track layouts]({{site.url}}/assets/images/docs/tools/devtools/track-layouts.png)
+
+### Track paints
+
+To see paint information in the timeline, enable the Track Paints option:
+
+![Screenshot of track paints]({{site.url}}/assets/images/docs/tools/devtools/track-paints.png)
+
+## Debugging options
+
```

Comment:
Oh, please add a sentence here. It's bad to have a level 2 header followed immediately by a level 3 header with no text in between. And it can make the website act squirrelly. (It's also bad for accessibility.) Oh, wait, you're just not done here.

Reply:
oops, I decided to follow up with section: https://github.com/flutter/website/issues/6750